src/pages/SpeciesCompendia/DownloadCompendia.js

```diff
@@ -0,0 +1,106 @@
+import Dropdown from '../../components/Dropdown';
+import Button from '../../components/Button';
+import { Ajax, formatSentenceCase } from '../../common/helpers';
+import { useLoader } from '../../components/Loader';
+import { Link } from 'react-router-dom';
+import Checkbox from '../../components/Checkbox';
+
+import { createToken } from '../../state/token';
+import uniq from 'lodash/uniq';
+import { push } from '../../state/routerActions';
+
+let DownloadCompendia = ({ token, createToken, push }) => {
+  const agreedToTerms = !!token;
+  const [selected, setSelected] = React.useState(null);
+  const [agree, setAgree] = React.useState(agreedToTerms);
+  const { data, isLoading } = useLoader(fetchCompendiaData);
+
+  if (isLoading || data.length === 0) return null;
+
+  return (
+    <div className="download-compendia">
+      <div>
+        <div className="download-compendia__title">Download the Compendia</div>
+
+        <div>
+          <span className="download-compendia__choose">Choose Organism</span>
+          <Dropdown
+            options={data}
+            selectedOption={selected}
+            label={formatSentenceCase}
+            onChange={setSelected}
+          />
+        </div>
+
+        <div className="download-compendia__row">
+          {!agreedToTerms && (
+            <div>
+              <Checkbox
+                name="termsOfService"
+                checked={agree}
+                onClick={() => setAgree(!agree)}
+              >
+                I agree to the{' '}
+                <Link
+                  to="/terms"
+                  className="link"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Terms of Use
+                </Link>
+              </Checkbox>
+            </div>
+          )}
+          <Button
+            text="Download Now"
+            className="download-compendia__button"
+            isDisabled={!agree}
+            onClick={async () => {
+              const tokenId = token || (await createToken());
+              const selectedOrganism = selected || data[0];
+              const downloadUrl = await downloadCompendia(
+                selectedOrganism,
+                tokenId
+              );
+              push({
+                pathname: '/species-compendia/download',
+                state: { organism: selectedOrganism, downloadUrl }
+              });
+            }}
+          />
+        </div>
+      </div>
+    </div>
+  );
+};
+DownloadCompendia = connect(
+  state => ({
+    token: state.token
+  }),
+  { createToken, push }
+)(DownloadCompendia);
+export default DownloadCompendia;
+
+async function fetchCompendiaData() {
+  try {
+    const data = await Ajax.get('/compendia');
+    return uniq(data.map(x => x.organism_name));
+  } catch (e) {
+    return [];
+  }
+}
+
+async function downloadCompendia(organism, token) {
+  // refetch the compendias, now sending the token id to retrieve the download urls
```

Comment:
I think `compendias` should be `compendia`, which is already the plural form of `compendium`.

Reply:
```suggestion
  // refetch the compendia, now sending the token id to retrieve the download urls
```